src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -87,14 +45,41 @@ public IndexModel describeIndex(String indexName) throws PineconeException {
         return indexModel;
     }
 
-    public IndexModel configureIndex(String indexName, ConfigureIndexRequest configureIndexRequest) throws PineconeValidationException {
-        if (configureIndexRequest == null) {
-            throw new PineconeValidationException("ConfigureIndexRequest object cannot be null");
-        }
+    public IndexModel configureIndex(String indexName, String podType, Integer replicas) throws PineconeValidationException {
         if (indexName == null || indexName.isEmpty()) {
-            throw new PineconeValidationException("Index name cannot be null or empty");
+            throw new PineconeValidationException("indexName cannot be null or empty");
+        }
+
+        // Set default values if parameters are not provided
+        String defaultPodType = "p1.x1";
+        int defaultReplicas = 1;
```

Comment:
Hmmm... I might be misunderstanding everything, but I don't I think... how else do we pass default values when the user only provides either `podType` or `replicas`?

Reply:
The user is either updating replicas, podType, or both. We wouldn't want to default any values for a configure / PATCH request because we may be overriding the existing value. Does that make sense?

I don't think these variables are actually being used anywhere in the code though, right?

Reply:
Ohhhh